packages/grafana-data/src/types/navModel.ts

```diff
@@ -1,4 +1,7 @@
 export interface NavModelItem {
+  /** @deprecated NavModelItems are translated, so use the messages from navBarItem-translations.ts
+   *  instead and use item.text as a fallback only
+   */
```

Comment:
Thoughts on adding a deprecation notice here, so we get this in editors? 

![image](https://user-images.githubusercontent.com/46142/150745880-c5f778e3-b174-4f70-9836-1cb75452c01d.png)

Reply:
hmmmm... idk, if we're never going to remove `text` it feels like a bit of a misuse of `deprecated`. but i can see both sides, it's a tough one 🤷

Reply:
I was on the fence about it - so this is enough to push me off.